recipes/openh264/all/conanfile.py

```diff
@@ -105,13 +98,14 @@ def package(self):
 
         if is_msvc(self) or self._is_clang_cl:
             rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
-            if self._preserve_dll_name:
-                # INFO: Preserve same old library name as when building with Make on Windows but using Meson
-                rename(self, os.path.join(self.package_folder, "lib", "openh264.lib"),
-                    os.path.join(self.package_folder, "lib", "openh264_dll.lib"))
-            else:
-                rename(self, os.path.join(self.package_folder, "lib", "libopenh264.a"),
-                    os.path.join(self.package_folder, "lib", "openh264.lib"))
+            if Version(self.version) <= "2.4.1":
+                if self.options.shared:
```

Comment:
@toge I made some change here to clarify the expression, can you confirm it is what you intended?

Reply:
@perseoGI 
Thank you for your improvements!
But there is a ci error. I'm try to solve it.

Reply:
Thats great! Thank you!